Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/configs.yaml

```diff
@@ -139,6 +140,36 @@ ValueAddNP:
       kvFunction: findAllWithKV
       where: enabled
 
+CancellationAndNoShowChargeConfigs:
+  tableName: cancellation_and_now_show_charge_configs
+
+  fields:
+    id : Id CancellationAndNoShowChargeConfigs
+    merchantId: Id Merchant
+    merchantOperatingCityId: Id MerchantOperatingCity
+    vehicleServiceTierType: ServiceTierType|NoRelation
+    currency: Currency
+    minCancellationCharge: HighPrecMoney
+    maxCancellationCharge: HighPrecMoney
+    perMetreCancellationCharge: HighPrecMoney
+    perMinuteCancellationCharge: HighPrecMoney
+    maxWaitingTimeAtPickupSeconds:  Seconds
+    freeCancellationTimeSeconds: Seconds
+    percentageOfRideFareToBeCharged: HighPrecMoney
```

Comment:
Why HighPrecMoney? It should be Double or Centesimal, right?

Reply:
Ya, edited